rosplan_planning_system/src/PlanDispatch/EsterelPlanDispatcher.cpp

```diff
@@ -104,6 +104,7 @@ namespace KCL_rosplan {
 		// begin execution
 		finished_execution = false;
 		state_changed = false;
+		bool plan_started = false;
```

Comment:
I'm not sure why this change is showing, since the commit is already in #126 (it's fine anyway)